### PR TITLE
Trilinos: fix source url

### PR DESCRIPTION
**Pull request by smoors**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 5 out of 6 (1 easyconfigs in this PR)
easybuild.novalocal - Linux centos linux 7.5.1804, Intel Xeon E312xx (Sandy Bridge), Python 2.7.5
See https://gist.github.com/20c1272a69faccff8437bb2d4ba2cc20 for a full test report.

**Comment by smoors** [comment]
ok, it looks like the source was significantly changed.
2 patches don't apply because the files they act on are not present anymore.
it's probably not a good idea to go forward with this?

**Comment by verdurin** [comment]
@smoors yes, looks like potentially a lot of work to fix. Up to you.

File: easybuild/easyconfigs/t/Trilinos/Trilinos-12.12.1-foss-2018a-Python-3.6.4.eb [hunk 1]
```diff
@@ -10,19 +10,16 @@ description = """The Trilinos Project is an effort to develop algorithms and ena
 toolchain = {'name': 'foss', 'version': '2018a'}
 toolchainopts = {'usempi': True, 'pic': True, 'strict': True}
 
-source_urls = [
-    'http://trilinos.csbsju.edu/download/files/',
-    'https://trilinos.org/oldsite/download/files/',
-]
-sources = ['%(namelower)s-%(version)s-Source.tar.gz']
+source_urls = ['https://github.com/trilinos/Trilinos/archive/']
```
[the remark this answers is not in the document]
Comment (smoors) [reply easybuild/easyconfigs/t/Trilinos/Trilinos-12.12.1-foss-2018a-Python-3.6.4.eb]: so the 'oldsite' is up again, that's great.
I can close this PR then.

**Pull request closed without merging**